From 7f204941ee01bcea4ab4a93e8a13ce8021f04dfa Mon Sep 17 00:00:00 2001
From: MSe1969 <mse1969@posteo.de>
Date: Fri, 9 Oct 2020 11:59:15 +0200
Subject: [PATCH] Avoid out-of-memory error during build

---
 gradle.properties | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gradle.properties b/gradle.properties
index 5bac8ac..0551412 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -1 +1,2 @@
 android.useAndroidX=true
+org.gradle.jvmargs=-Xmx2048m -XX:+HeapDumpOnOutOfMemoryError
-- 
2.11.0

